packages/roosterjs-editor-types/lib/interface/ClipboardData.ts

```diff
@@ -31,6 +31,11 @@ export default interface ClipboardData {
      */
     image: File | null;
 
+    /**
+     * General file from clipboard event
+     */
+    files: File[];

```

Comment:
please make this optional otherwise it will be a breaking change.

Reply:
Good catch. Made it optional.